Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -829,9 +829,9 @@ static void dispatchSyntheticMouseMove(LocalFrame& mainFrame, const WebCore::Flo
         auto* localMainFrame = dynamicDowncast<LocalFrame>(m_page->mainFrame());
```

Comment:
Should be: `RefPtr localMainFrame = dynamicDowncast<LocalFrame>(m_page->mainFrame());`

Reply:
Done.